src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateNotificationService.swift

```diff
@@ -112,6 +112,38 @@ class HealthCertificateNotificationService {
 		}
 	}
 
+	func scheduleCertificateReissuanceNotificationIfNeeded(
+		for person: HealthCertifiedPerson,
+		previousCertificateReissuance: DCCCertificateReissuance?,
+		completion: (() -> Void)? = nil
+	) {
+		let name = person.name?.standardizedName
+		guard let newCertificateReissuance = person.dccWalletInfo?.certificateReissuance else {
+			Log.info("No certificate reissuance found for person \(private: String(describing: name))", log: .vaccination)
+			completion?()
+
+			return
+		}
+
+		if newCertificateReissuance != previousCertificateReissuance {
+			// we need to have an ID for the notification and since the certified person doesn't have this property "unlike the certificates" we will compute it as the hash of the string of the standardizedName + dateOfBirth
```

Comment:
Would it be possible to introduce that identifier to the person? (could be the same, hash of name and dateOfBirth)

Reply:
Yes. I found four places where we calculated the hash independently, so I've added the identifier to the person now